<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalDimensions</class>
 <widget class="QDialog" name="FinalDimensions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>160</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/room_size_2d_min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>171</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/depth_min.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LeftVertSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>20</width>
     <height>133</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TopHorSize">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>160</y>
     <width>157</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/horizontal_dimension.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BottomLeftSize">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/arrows-diagonal.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BottomRightSize">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/arrows-diagonal_1.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BottomCenterSize">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>18</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension_2.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BottomCenterSize_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>21</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/vertical_dimension_2.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showVertSize">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="showHorSize">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>65</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>65</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>193</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>573</width>
     <height>573</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/proj/img/title.png);
opacity: 50;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>165</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Месела нижняя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>237</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>208</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Месела верхняя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>278</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Мезуза</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Нижняя и верхняя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>318</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>331</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Створка высота</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>373</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Створка высота с зацепом</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>414</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Альбаша 57 круглая</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>477</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Cтекло прозрачное</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>503</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>193</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>237</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>318</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_15">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>278</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>503</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>193</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>237</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>318</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_20">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>465</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_21">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_22">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_23">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>278</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_25">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>193</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1210</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_26">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>237</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1160</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_27">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>318</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>674</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_29">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>654</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_30">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>683</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_31">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>278</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>748</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_32">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_34">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>465</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_35">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>465</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_36">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>465</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>437</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_37">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>528</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_38">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>528</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>528</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60,230,150);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Алюминий</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>LeftVertSize</zorder>
  <zorder>TopHorSize</zorder>
  <zorder>BottomLeftSize</zorder>
  <zorder>BottomRightSize</zorder>
  <zorder>BottomCenterSize</zorder>
  <zorder>BottomCenterSize_2</zorder>
  <zorder>showVertSize</zorder>
  <zorder>showHorSize</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label_13</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>label_14</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>lineEdit_8</zorder>
  <zorder>lineEdit_9</zorder>
  <zorder>lineEdit_10</zorder>
  <zorder>lineEdit_11</zorder>
  <zorder>lineEdit_12</zorder>
  <zorder>lineEdit_13</zorder>
  <zorder>lineEdit_14</zorder>
  <zorder>lineEdit_15</zorder>
  <zorder>lineEdit_16</zorder>
  <zorder>lineEdit_17</zorder>
  <zorder>lineEdit_18</zorder>
  <zorder>lineEdit_19</zorder>
  <zorder>lineEdit_20</zorder>
  <zorder>lineEdit_21</zorder>
  <zorder>lineEdit_22</zorder>
  <zorder>lineEdit_23</zorder>
  <zorder>lineEdit_25</zorder>
  <zorder>lineEdit_26</zorder>
  <zorder>lineEdit_27</zorder>
  <zorder>lineEdit_29</zorder>
  <zorder>lineEdit_30</zorder>
  <zorder>lineEdit_31</zorder>
  <zorder>lineEdit_32</zorder>
  <zorder>lineEdit_34</zorder>
  <zorder>lineEdit_35</zorder>
  <zorder>lineEdit_36</zorder>
  <zorder>lineEdit_37</zorder>
  <zorder>lineEdit_38</zorder>
  <zorder>label_17</zorder>
  <zorder>lineEdit_38</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
